contracts/AssetPool.sol

```diff
@@ -478,6 +478,8 @@ contract AssetPool is Ownable, IAssetPool {
         } else {
             toMint = (amount * covSupply) / collateralBalance;
         }
+        require(toMint > 0, "Minted amount would be 0");
```

Comment:
How about "Minted tokens amount must be greater than 0"?

Reply:
Alternative solution but probably an overkill is to keep track of all collateral tokens through `deposit()` and `withdrawal()` functions. Even if someone transfer `collateralToken` directly, it won't affect the calculation around `toMint` tokens.

Reply:
Fixed in 1112ce07ad93245fc41dd9c69431d2b9c6e9a1fe.

Reply:
As for alternative solution (keeping track of all collateral tokens in `asset pool` via `deposited` and `withdrawal`:
I thought about, but came to the conclusion that it would indeed be an overkill.